context/app/routes_cells.py

```diff
@@ -65,26 +68,72 @@ def timed(*args, **kwargs):
 
 
 @timeit
+def preload_cells_api(app):
+    # Preload the gene symbols, protein IDs, and cell IDs on server startup
+    # so that they are immediately available when the user starts typing.
+
+    funcs = [_get_gene_symbols, _get_protein_ids, _get_cell_ids]
+
+    for func in funcs:
+        func(app)
+
+
+@timeit
+@cache
```

Comment:
The `cache` persists for as long as the server runs.

I used the `@cache` approach since this was what Chuck said we should do in a TODO he left before we used Python 3.9.

Flask-Caching does look like a possible future improvement, but is also a bigger lift to integrate. `@cache` was more similar to the global dict approach that was already implemented, so I stuck with it.